scalableminds/webknossos
# Zoom value is the same scale in flight mode

### Mailable description of changes (needs to be understandable by webknossos mailing list people):
- Zoom scale in ortho and flightmode are equal

### Steps to test:
- Open a skeleton tracing and make ortho and flight viewports same size (with k,l)
- Verify that the zoom scale is equal

### Issues:
- fixes #1829

------
